### PR TITLE
feat: upgrade openedx-event-sink-clickhouse to 0.2.0

### PR DESCRIPTION
**Pull request by Ian2012**
This PR upgrade openedx-event-sink-clickhouse to 0.2.0.

It also automatically creates waffle flags for every model in event_sink so that those will be enabled by default

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0015_event_sink_user_profile.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            state String NOT NULL,
+            goals String NOT NULL,
+            bio String NOT NULL,
+            profile_image_uploaded_at String NOT NULL,
+            phone_number String NOT NULL,
+            dump_id UUID NOT NULL,
+            time_last_dumped String NOT NULL
+        ) engine = MergeTree
+        PRIMARY KEY (id, time_last_dumped)
+        ORDER BY (id, time_last_dumped);
```
Comment (Ian2012) [inline tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0015_event_sink_user_profile.py:39]: @SoryRawyer can you take a look at this table?

**Comment by Ian2012** [comment]
@bmtcril not sure why that error is happening. Not sure if it's the profile or if it's an error in Redis.

I've tested this PR in three different local environments. Already initialized, empty environments, and in none of these I was able to reproduce this error.

File: tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0015_event_sink_user_profile.py (hunk 1 above)
Comment (SoryRawyer) [reply tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0015_event_sink_user_profile.py:39]: @Ian2012 could you point me to the source table for this? I'm not familiar with the process that creates this data.

Comment (Ian2012) [reply tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0015_event_sink_user_profile.py:39]: Sure, there is the PR that added that functionality for the `openedx-event-sink-clickhouse` plugin https://github.com/openedx/openedx-event-sink-clickhouse/pull/28

**Comment by bmtcril** [comment]
@Ian2012 it looks like redis isn't a requirement for the LMS init job, so it will work if it's already running but I don't see it pulling or starting the container here. If you stop everything then run the init on a clean environment does it still happen?

**Review by bmtcril — APPROVED** [review]
👍

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-aspects&entry.867055334=https://github.com/openedx/tutor-contrib-aspects/pull/340&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-08-30+14:19&entry.1917517419=2023-09-07+13:33&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**